operations/mimir/autoscaling.libsonnet

```diff
@@ -137,21 +139,27 @@
     },
   },
 
-  local siToBytes(str) = (
+  local siToBytesString(str) = (
```

Comment:
From a design perspective, I think this should be a generic utility returning a number and then if the call needs a string it will cast it to string (which is what were were doing before).

Reply:
@pracucci actually before it could return either a number or a string.

Reply:
@pracucci I converted it now into returning an integer.